tsconfig.base.json

```diff
@@ -12,9 +12,6 @@
     "module": "esnext",
     "moduleResolution": "node",
     "noFallthroughCasesInSwitch": true,
-    "paths": {
-      "react-hyperscript-helpers": ["./types/react-hyperscript-helpers"]
-    },
```

Comment:
Removing this from the base package config. Individual packages can add it if they use React Hyperscript Helpers.

Reply:
so... shouldn't we see files changes for those packages that still need react-hyperscript-helpers?

Reply:
Components was the only package that depended on react-hyperscript-helpers.

Reply:
ah, ok